backend/app/services/courses/show.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Courses
+  class Show < Core::Service
+    def call(params:)
+      @params = params
+      @id = (params[:id]).delete(':')
+      begin
+        course = Course.find(id)
+      rescue ActiveRecord::RecordNotFound
+        error = "Couldn't find Course with 'id'=#{id}"
+      end
```

Comment:
Maybe better to find a course using the method find_by_id, to handle nil instead of rescue error?

Reply:
Yeah, you right. This will make code more clear. Thank you!